test/ngMock/angular-mocksSpec.js

```diff
@@ -2408,9 +2408,134 @@ describe('ngMockE2E', function() {
   });
 });
 
+
 describe('make sure that we can create an injector outside of tests', function() {
   //since some libraries create custom injectors outside of tests,
   //we want to make sure that this is not breaking the internals of
   //how we manage annotated function cleanup during tests. See #10967
   angular.injector([function($injector) {}]);
 });
+
+
+describe('don\'t leak memory between tests', function() {
```

Comment:
I prefer describe block descriptions to be nouns of things that are being tested...

But I see you are following the pattern above :-(